jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/NamedObjectItemPresenter.java

```diff
@@ -102,6 +116,25 @@ public void signalParameterAddedOrRemoved() {
         fireChangeEvent();
     }
 
+    public void openEditModal() {
+        ItemObjectModel itemObjectModel = new ItemObjectModel();
```

Comment:
not sure I understood why you need to create another _ItemObjectModel_ as it set in model attribute already.

Reply:
yes